addons/laser/stringtable.xml

```diff
@@ -5,6 +5,7 @@
             <English>Laser Dispersion Simulation Count</English>
             <Japanese>レーザーの分散シミュレート数</Japanese>
             <German>Laserstreuung-Simulationszähler</German>
+            <Polish>Wskaźnik poziomu rozproszenia wiązki lasera </Polish>
```

Comment:
yes trailing space is "by accident" :)
And i'll probably have to find much better translations,, for this phraze